pkg/scaffold/project/gitignore.go

```diff
@@ -52,7 +52,7 @@ bin
 *.out
 
 # Kubernetes Generated files - skip generated files, except for vendored files
-
+**/zz_generated.*
```

Comment:
I think we had this earlier and had to specifically take it out because without generated files checked-in, project will not be in compilable state which is a problem for projects that needs to be imported using `go get`.

Reply:
Hmmmm yeah a compilable state would be important for `go get`, that's a great point. I'm currently working through the kubebuilder book and piecing things together, so my question is mostly naive, but isn't some of the code generation from `// +` declarations done at `go get` build time? Meaning two types of generations (commit vs not) where scaffolding has files to check in but build time stuff shouldn't be checked in?

Reply:
>but isn't some of the code generation from // + declarations done at go get build time

No. Makefile has a `generate` target that does the code generation.  `generate` is invoked as pre-requisite for other targets.

We used to use `go: generate` directive in project version "1" and that would trigger code generation at `go build`, but we realized it was inflexible and was also hiding too much from the users, so we took it away.

IMO, this change is not needed (feel free to close this PR).

Reply:
I appreciate the comparison to v1, thank you. I'll close this :+1: